android/README.md

```diff
@@ -0,0 +1,75 @@
+# Salesforce Mobile SDK for Android Package
+
+The **forcedroid** npm package allows users to create Android mobile applications to interface with the [Salesforce Platform](http://www.salesforce.com/platform/overview/), leveraging the [Salesforce Mobile SDK for Android](https://github.com/forcedotcom/SalesforceMobileSDK-Android).
+
+## Getting Started
+
+If you're new to mobile development, or the force.com platform, you may want to start at the [Mobile SDK landing page](http://wiki.developerforce.com/page/Mobile_SDK).  This page offers a variety of resources to help you determine the best technology path for creating your app, as well as many guides and blog posts detailing how to work with the Mobile SDK.
+
+But assuming you're all read up, here's how to get started with the **forcedroid** package to create the starting point for your mobile application.
+
+## Install the forcedroid Package
+
+Because forcedroid is a command-line utility, we recommend installing it globally, so that it's easily accessible on your path:
+
+        sudo npm install forcedroid -g
+
+You're of course welcome to install it locally as well:
+
+        npm install forcedroid
+
+In this case, you can access the forcedroid app at `[Install Directory]/node_modules/.bin/forcedroid`.
+
+## Using forcedroid
+
+For the rest of this document, we'll assume that `forcedroid` is on your path.
+
+Typing `forcedroid` with no arguments gives you a breakdown of the usage:
+
+        $ forcedroid
```

Comment:
You can go ahead and merge, and I can send a PR with the changes to the README.md.